tools/disasm/files.txt

```diff
@@ -8,7 +8,7 @@
 #
 
 [
-    ['makerom', 'extracted/n64-us/baserom/', 'makerom',
+    ['makerom', '$(BASEROM)', 'makerom',
```

Comment:
this change doesn't seem appropriate given that the data within this file is version specific, and providing a `$(BASEROM)` of a different version string would be incorrect.

Reply:
well yes it's implied the baserom folder from arguments should contain the segments for the right version

Reply:
The way it looks to me makes it seem like `input path` isn't only specifying the rom path, it also associates the data to a specific version (n64-us). It makes no sense to replace `$(BASEROM) with `extracted/n64-jp/baserom/` for example because the data in this file won't be applicable to that version.

In fact, I don't understand why we're even storing the `$(BASEROM)` path here to begin with. Shouldn't disasm.py just be handling that internally now?

Reply:
I guess this is a fair point. The paths of every entry is the same so it would be fine to yeet it.
I'm not sure how many changes this would require tho

Reply:
If you do the bare minimum it should just be a one line change (replace the only `segment[1]` reference with `baserom_segments_dir`). I wouldn't do much more than that in this PR (unless you feel motivated to fix everything right now) since disasm.py looks like it needs to be cleaned up a bit to support multi-version anyway

Reply:
This really depends on how we decide to deal with disassembling in multi-version, which will require discussion when we get to it. I would think we would use spimdisasm (which uses a different config system anyways). Tbh I would say my preferred option would be just revert the disasm changes, however doing the bare minimum mzx suggests or even just leaving as is I would be fine with.

Reply:
Replaced `'$(BASEROM)'` with None and it'sn ow unused.
I didn't remove it because disasm.py is very spaghetti in its usage of this data, would need to offset dozens of indexing by 1